onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -1264,12 +1351,17 @@ def _capture_graph_and_evaluate_torch_script_evaluator(function: Callable, args,
                 f"Model:\n"
                 f"{onnxscript.proto2text(onnx_model)}"
             ) from e
-        # Disable all ORT optimizations
-        session_options = onnxruntime.SessionOptions()
-        session_options.graph_optimization_level = (
-            onnxruntime.GraphOptimizationLevel.ORT_DISABLE_ALL
-        )
+
         try:
+            if os.environ.get("CATCH_ORT_SEGFAULT"):
+                # Use an individual process to run ONNX Runtime to catch segfaults
+                return _safe_ort_session_run(onnx_model.SerializeToString(), ort_inputs)
+
+            # Disable all ORT optimizations
+            session_options = onnxruntime.SessionOptions()
```

Comment:
nit: this part is repeated

Reply:
It is. Thoughts on combining? I felt it was not worth breaking down `_ort_session_run` at the time because it also handles signals

Reply:
Merged with _ort_session_run